app/src/pages/inside/launchesPage/LaunchToolbar/actionPanel/actionPanel.scss

```diff
@@ -22,3 +22,48 @@
     display: none;
   }
 }
+
+.counter {
+  position: absolute;
+  right: 3px;
+  bottom: 50%;
+  box-sizing: border-box;
+  min-width: 15px;
+  height: 15px;
+  padding: 2px 4px;
+  margin: 0 0 5px 1px;
+  border: 1px solid $COLOR--white-two;
+  border-radius: 100px;
+  background-color: $COLOR--orange-red;
+  color: $COLOR--white-two;
+  font-size: 10px;
+  line-height: 10px;
+  text-align: center;
+  animation: bounce 2s infinite;
```

Comment:
Maybe it's better to make period between bounces longer? Could you please discuss this moment with UX designer?

Reply:
I discussed this moment with UX designer. He is ok with it